dell/csi-powerscale
# Installation using centralize helm

# Description
Installation using centralize helm

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/877|

# Checklist:

- [X] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration
  steps to test 
```
  cd  csi-powerscale\dell-csi-helm-installer
  wget -O myValues.yaml  https://github.com/dell/helm-charts/raw/csi-isilon-2.7.0/charts/csi-isilon/values.yaml
  ./csi-install.sh --namespace isilon --values myValues.yaml
```
- [X] installed using script file 
![image](https://github.com/dell/csi-powerscale/assets/109663924/9fd662b5-36de-4cba-81a2-425ee14cc99f)

make check result
![image](https://github.com/dell/csi-powerscale/assets/109663924/5479bdbb-27bf-4ac9-824c-6e48d9f200a6)

code coverage 
![image](https://github.com/dell/csi-powerscale/assets/109663924/735b966b-5331-45b6-894e-23e0fdcc43f7)
